Formula/octave.rb

```diff
@@ -63,6 +65,11 @@ def install
     # cause linking problems.
     inreplace "src/mkoctfile.in.cc", /%OCTAVE_CONF_OCT(AVE)?_LINK_(DEPS|OPTS)%/, '""'
 
+    args = []
+    if build.without? "qt"
+      args << "--without-qt"
+    end
```

Comment:
Write the `if` statement as a one-liner.

```
args = []
args << "--without-qt" if build.without? "qt"
```

Reply:
Much better, thanks! I was thinking I could do something like that, but my Ruby is rusty.